blog/content/post/2019-10-24-meta-pcengines-xen.md

```diff
@@ -0,0 +1,128 @@
+---
+title: 'XEN on the PC Engines apu2 - why you want to use it'
+abstract: 'Xen Project creates a software system that allows the execution of multiple virtual guest operating systems simultaneously on a single physical machine. In this case, it is a PC Engines apu2 platform. '
+cover: /covers/3mdeb-logo_simple-color.png
+author: norbert.kaminski
+layout: post
+published: false
+date: 2019-10-24
+archives: "2019"
+
+tags:
+  - XEN
+  - meta-pcengines
+  - apu2
+  - PC engines
+categories:
+  - OS Dev
+  - Security
+  - Manufacturing
+
+---
+# What is the Xen Project?
+
+Xen Project creates a software system that allows the execution of multiple virtual guest operating systems simultaneously on a single physical machine.
+It is valuable when you want to create a complex security software system.
+None of the guest operating systems have direct access to the host hardware.
+
+A Xen hypervisor manages the virtual machine guests known as domains.
+It is controlled by a special domain called domain0 (dom0). Only the hypervisor
+has access to the physical outputs of the platform. Depending on the guest type
+(HVM or PV) hypervisor allows unprivileged domains (domUs) to use virtual
+hardware devices or passes the links to the physical hardware. To know more look
+at the Xen Project [wiki site](https://wiki.xenproject.org/wiki/Xen_Project_Beginners_Guide).
+
+<i>
+    <h2><p align=center>Xen Project architecture </p>
+</i>
+
+![Diagram](https://cloud.3mdeb.com/index.php/s/HNmc8yqbcCQzCje/preview)
+
+# How to build the dom0 on the PC Engines APU2?
+
+For this purpose, we will use the [meta-pcengines repository](https://github.com/pcengines/meta-pcengines.git).
```

Comment:
You should mention which commit was used for  this blog post. The current master branch will look differently.